tests/testthat/testFunction.R

```diff
@@ -49,3 +49,25 @@ is_identical_without_name <- function(x, y,maxLength=5){
 	
 	return(ret);
 	}
+
+
+prepare.test.cases <- function(venn.test) {
+    for (i in 1:length(venn.test)) {
+    	for (j in 1:length(venn.test[[i]])) {
+    	    grob.type <- class(venn.test[[i]][[j]])[1];
+    	    
+    		if (grob.type == 'polygon') {
+    		    # Strip polygons of their x and y values
+    	        # This is also included in the params field
+    			venn.test[[i]][[j]]$x <- NULL;
+    			venn.test[[i]][[j]]$y <- NULL;
+    		} else if (grob.type == 'text') {
+    		    # Strip text of duplicate fontface value
+    		    # for backwards compatbility
+    		    venn.test[[i]][[j]]$gp$fontface <- NULL;
+        		}
```

Comment:
Adjust the tab alignment to match ratliff style (here and a on couple other lines).

Reply:
I'm pretty sure this is the correct indentation for `else if` (only adding the extra indent for the last closing bracket. What would you change it to?

Reply:
> As this submission is time sensitive, I'm going to merge now.

Fair. :)

> I'm pretty sure this is the correct indentation ...

Looks like the misalignment is due to tabs instead of spaces. I replaced `\t` with four spaces and it looks fine.